src/integrator_whfast.c

```diff
@@ -731,19 +734,17 @@ static void integrator_apply_corrector(double inv){
 	}
 }
 
-
-void integrator_whfast_part1(void){
+void integrator_whfast_part1(){
 	gravity_ignore_10 = 1;
-	unsigned int recalculate_jacobi = (!integrator_whfast_synchronize_manually) || integrator_whfast_particles_modified;
 	if (integrator_allocated_N != N){
 		integrator_allocated_N = N;
 		p_j = realloc(p_j,sizeof(struct particle)*N);
 		eta = realloc(eta,sizeof(double)*(N-N_megno));
 		etai= realloc(etai,sizeof(double)*(N-N_megno));
-		recalculate_jacobi = 1;		// Recalculate masses/Jacobi coordinates if first timestep or if N changes.
+		integrator_whfast_particles_modified = 1;
 	}
 	// Only recalculate Jacobi coordinates if needed
-	if (recalculate_jacobi){
+	if (integrator_whfast_recalculate_jacobi_every_timestep || integrator_whfast_particles_modified){
```

Comment:
Again, I think it would make sense to rename `integrator_whfast_particles_modified` to `integrator_whfast_recalculate_jacobi_this_timestep`.